tools/WixBuild.props

```diff
@@ -166,8 +172,10 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <BuildARM>$(WindowsSDKDesktopARMSupport)</BuildARM>
+    <BuildARM Condition=" Exists('$(PlatformSdkArmLibraryRoot)') ">true</BuildARM>
     <BuildARM Condition=" '$(BuildARM)'!='true' ">false</BuildARM>
+    <BuildARM64 Condition=" Exists('$(PlatformSdkArmLibraryRoot)') ">true</BuildARM64>
+    <BuildARM64 Condition=" '$(BuildARM64)'!='true' ">false</BuildARM64>
```

Comment:
I didn't see anything using this as a condition to skip building, did I miss something?

Reply:
Nope. Probably should have a couple of those...

Reply:
Ugh. Detection sucks. We need ARM64 in both the Windows SDK (included by default) and VS2017. We'd need VSWhere to determine if the ARM64 tools and libs are installed. We could also require a property to opt-in to the build, so you're not surprised when it fails on ARM/ARM64. @robmen, @rseanhall, opinions?

Reply:
I think requiring a property to opt-in is perfectly fine.

Reply:
I'm good with opt-in property as long as the OFFICIAL_BUILD fails if the opt-in property is not set. That way we don't forget to set in in the official build (because I undoubtedly will).